### DISCUSSION TITLE
reduce the number of requests

### DISCUSSION THREAD

**Discussion by Lengso** [projectdiscovery/uncover#10] [Ideas]
According to the [**uncover** document](https://github.com/projectdiscovery/uncover/blob/main/README.md#running-uncover), you can search for multiple dorks
 
My idea is to read multiple statements in the dork.txt file, which can be combined together, and one request can get the return results of all statements

dork.txt
```
ssl:"Uber Technologies, Inc."
title:"Grafana"
```

For example, all engines support `|| or` flag
```
fofa :  ssl:"Uber Technologies, Inc." || title:"Grafana"
censys : ip="1.1.1.1" or  ip="1.1.1.1"
```

**Comment by ehsandeep** [comment]
@Lengso it is an example of using a file input for multiple dorks when required, we can still combine our query as part of the single query for the supported search engine, what changes do you suggest in this case?

**Comment by Lengso** [comment]
@ehsandeep This is a way of thinking, I think it is feasible but I have to make some judgments, because the flags of each engine combination are different
